recipes/ggcaller/meta.yaml

```diff
@@ -50,17 +66,29 @@ requirements:
     - xorg-libxpm
     - xorg-libxaw
     - xorg-libxfixes
+    - intbitset
+    - networkx
+    - uncertainties
+    - eigen 3.3
+    - tqdm
+    - seaborn
+    - rapidnj
+    - python-edlib
+    - joblib
+    - gffutils
+    - cd-hit
+    - python-wget
 
 test:
   imports:
     - ggCaller_cpp
-    - balrog
     - ggCaller
+    - panaroo_runner
```

Comment:
Double-checking this name doesn't clash with anything from panaroo?

Reply:
I've checked the panaroo github and recipe on bioconda, no clashes that I could find.